CONTRIBUTORS.md

```diff
@@ -80,6 +80,20 @@ Deepak suryawanshi
 &#47;&gt;</a>
 </td>
 
+<td>
+<a href="https://www.linkedin.com/in/aaliya7516" >
+<img src="https://avatars1.githubusercontent.com/u/53328555?v=4" alt="Aaliya Haque" title="Aaliya Haque" width="100px" height="auto" />
```

Comment:
Can you please change the image url to your GitHub avatar url?

Reply:
yeah sure